src/arguments/CoreFloat.ts

```diff
@@ -1,37 +1,27 @@
 import type { PieceContext } from '@sapphire/pieces';
 import { Identifiers } from '../lib/errors/Identifiers';
+import { resolveFloat } from '../lib/resolvers';
 import { Argument, ArgumentContext, ArgumentResult } from '../lib/structures/Argument';
 
 export class CoreArgument extends Argument<number> {
+	private readonly messages = {
+		[Identifiers.ArgumentFloatTooSmall]: ({ minimum }: ArgumentContext) => `The argument must be greater than ${minimum}.`,
+		[Identifiers.ArgumentFloatTooBig]: ({ maximum }: ArgumentContext) => `The argument must be less than ${maximum}.`,
+		[Identifiers.ArgumentFloatError]: () => 'The argument did not resolve to a valid floating point number.'
```

Comment:
```suggestion
		[Identifiers.ArgumentFloatTooSmall]: ({ minimum }: ArgumentContext) => `The given number must be greater than ${minimum}.`,
		[Identifiers.ArgumentFloatTooLarge]: ({ maximum }: ArgumentContext) => `The given number must be less than ${maximum}.`,
		[Identifiers.ArgumentFloatError]: () => 'The argument did not resolve to a valid number.'
```

And update `resolveFloat` accordingly

p.s. non devs don't know what floating points are so don't include it in the message.

Reply:
All floating-point numbers are numbers, but not all numbers are floating-point numbers. I'd suggest changing it to "a valid decimal" (or "decimal numeral" more accurately, but that's rarely used).